Doc/library/ipaddress.rst

```diff
@@ -219,9 +219,14 @@ write code that handles both IP versions correctly.  Address objects are
       ``"::abc:7:def"``.
    2. An integer that fits into 128 bits.
    3. An integer packed into a :class:`bytes` object of length 16, big-endian.
+   4. String IPv6 address may contain scope (or zone) id in format <address>%<scope_id>.
+      Scope id could not be blank or negative integer.
+      See :RFC:`4007` for details.
 
    >>> ipaddress.IPv6Address('2001:db8::1000')
    IPv6Address('2001:db8::1000')
+   >>> ipaddress.IPv6Address('2001:db8::1000%1')
+   IPv6Address('2001:db8::1000%1')
```

Comment:
Is this example actually well-formed? If I'm understanding the RFC right, this example and the one above it can't actually both be valid (even if the module doesn't comprehensively validate such things and so the transcript is accurate.)

For the first example to be valid, `2001:db8::1000` must be a global address.

But in that case it's inappropriate to attach a scope zone ID to it. That only makes sense for non-global addresses. So a better example would be one from the RFC, like `ff02::5678%5`.

Reply:
I agree with you. As it is not prohibited by [RFC](https://tools.ietf.org/html/rfc4007#section-11.1), it could bring some misunderstanding. Will be fixed.